### DISCUSSION TITLE
Use Vault without Password / Encryption ?

### DISCUSSION THREAD

**Discussion by EVOTk** [Eugeny/tabby#7976] [Q&A]
Hello,
I'm use Tabby in a portable edition in an encrypted partition.
Is it possible to use vault, but without password to unlock? 

I don't need the encryption feature.

Thx

**Comment by TooMuhtsh** [comment]
There's now a middle ground: keep the encryption, skip the typing. tabby-better-vault unlocks the vault automatically via the OS keychain (DPAPI on Windows, Secret Service on Linux) — the vault stays encrypted, but you never type the master password on your own machine. Opt-in, on npm as tabby-better-vault.
